packages/fxa-settings/src/components/Settings/ProgressBar/index.tsx

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { useFtlMsgResolver } from '../../../models';
+
+export type ProgressBarProps = {
+  numberOfSteps: number;
+  currentStep: number;
+};
+
+export const ProgressBar = ({
+  numberOfSteps,
+  currentStep,
+}: ProgressBarProps) => {
+  const ftlMsgResolver = useFtlMsgResolver();
+  const localizedProgressBarAriaLabel = ftlMsgResolver.getMsg(
+    'progress-bar-aria-label',
+    `A progress bar showing that the user is on step ${currentStep} of ${numberOfSteps}.`,
+    { currentStep, numberOfSteps }
+  );
+  // If you have more than 6 steps, you may need to add custom widths into tailwind.
+  // https://v2.tailwindcss.com/docs/width#width-scale
+  let width =
+    currentStep === numberOfSteps
+      ? 'w-full'
+      : `w-${currentStep}/${numberOfSteps}`;
+  return (
+    <div
+      role="progressbar"
+      aria-label={localizedProgressBarAriaLabel}
```

Comment:
Nice work on adding all the accessibility attributes for the progressbar!! I learned something new today :)

I wasn't sure if the aria-label would get picked up because this is a non-interactive element, but Deque indicates its good 👍 🎉 
https://dequeuniversity.com/rules/axe/4.1/aria-progressbar-name

Reply:
@millsoper 
We might still need to add `role='progressbar'` for JAWS screen reader, but an aria-label might not be needed with the semantic `progress` element according to this: 
https://accessible-app.com/pattern/vue/progress
Not too sure about the focus management, to ensure that the steps are actually read out 🤔 

(All this complexity could be avoided if we got the green light from UX 🟢 to add "Step 1  of 4" back into the UI 😉 )